<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addanimalpage</class>
 <widget class="QDialog" name="addanimalpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>640</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter animal information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Breed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Age: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lifespan:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string># of Past Owners:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time to Dedicate per Week (hr):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimated Cost Per Week:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight (lb):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (cm):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBreed">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAge">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLifespan">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineOwners">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCost">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>140</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineWeight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineHeight">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="Noise">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Noise Level:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Noise_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Noise_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Noise_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Noise_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Noise_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Intelligence">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Intelligence:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Intelligence_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Intelligence_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Intelligence_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Intelligence_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Intelligence_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Adapt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Adaptability:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Adapt_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Adapt_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Adapt_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Adapt_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Adapt_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Reproductability">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Reproductability:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Reproduce_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Reproduce_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Reproduce_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Reproduce_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Reproduce_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Appetite">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Appetite:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Appetite_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Appetite_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Appetite_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Energy">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Energy Level:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Energy_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Energy_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Energy_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Energy_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Energy_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Personality">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Personality:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Introvert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Introvert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Extrovert">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Extrovert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Ambivert">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ambivert</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Diet">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Diet:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Herbivore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Herbivore</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Carnivore">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Carnivore</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Omnivore">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Omnivore</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Environment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Environment:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Indoor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Indoor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Outdoor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Outdoor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Both">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Both</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Medication">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Requires Medication:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_True">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>True</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_False">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>False</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Gender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender:</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Male">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Female">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Other">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addanimalpage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addanimalpage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
